src/test/java/steps/ResponsesSteps.java

```diff
@@ -0,0 +1,28 @@
+package steps;
+
+import cucumber.api.java.en.Then;
+import org.junit.Assert;
+
+public class ResponsesSteps {
+
+    private ResourcesSteps projectSteps;
+
+    public ResponsesSteps(ResourcesSteps projectSteps) {
+       this.projectSteps = projectSteps;
+    }
+
+    @Then("^The response should contain the data with the string value \"(.*?)\" and \"(.*?)\"$")
+    public void theResponseShouldContainSentDataWithStringValue(String tag, String tagValue) throws Throwable {
```

Comment:
is not necessary, is not been used, we are not catch any exception/error I will remove.